plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -394,6 +394,7 @@ class TaskTable extends React.Component {
         sortBy={{ prop: "updated", order: "desc" }}
         sortOrder="desc"
         sortProp="updated"
+        uniqueProperty="id"
```

Comment:
Correct @nLight 

During my tests the `ids` were all unique with hundreds of tasks, the reason I opened the PR.